parity/migration.rs

```diff
@@ -215,6 +219,12 @@ fn exists(path: &Path) -> bool {
 	fs::metadata(path).is_ok()
 }
 
+// in-place upgrades that do nothing when called repeatedly
+fn run_inplace_upgrades(path: &Path) -> Result<(), Error> {
+	try!(migrations::upgrade_account_bloom(path));
+	Ok(())
```

Comment:
it's perfectly safe to run this migration any time on the upgraded database - it just won't do anything

any further upgrades should just go after this line

i don't want to introduce any new logic to the migration _in beta_ and at _this pr_